ci/h2spec.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+set -eu
+set -o pipefail
+shopt -s extglob
+
+SEP="$(tput bold)⣷$(tput sgr0)"
+LOGFILE="l5d-h2spec.log"
```

Comment:
consider `LOGFILE =$(mktemp -t l5d-h2spec.log)`, to avoid gitignore and overwriting.